script/openqa-load-templates

```diff
@@ -104,53 +104,49 @@ sub print_error ($res) {
     if (my $err = $res->error) {
         if (my $json = $res->json) {
             if (my $json_err = $json->{error}) {
-                die "$err->{message}: " . join("\n", @{$json_err})
-                  if ref $json_err eq 'ARRAY';
-                die "$err->{message}: $json_err";
+                die "$err->{message}: " . (ref $json_err eq 'ARRAY' ? join("\n", @{$json_err}) : $json_err);
```

Comment:
small wondering! maybe `wantarray` is suitable here, if you like https://perlmaven.com/wantarray

Reply:
I don't see the applicability